### PR TITLE
Add 4.10->4.11 upgrade test for azure

### PR DESCRIPTION
**Pull request by shellyyang1989**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 1]
```diff
@@ -223,6 +223,42 @@ tests:
...
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @azure-ipi'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-workers-marketimage
+- as: azure-upi-disconnected
+  cron: 0 17 */3 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
```
[the remark this answers is not in the document]
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml:243]: Thanks for the correction! Updated.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 2]
```diff
@@ -223,6 +223,42 @@ tests:
...
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.10 and @azure-upi'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @azure-upi'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test
```
[the remark this answers is not in the document]
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml]: ah, right. Updated. Thanks!

**Comment by kuiwang02** [comment]
lgtm for golang part

**Comment by shellyyang1989** [comment]
cc @liangxia for review and approval. Thanks!

**Pull request merged**